test/expr/postfix/dot/dot_keywords.swift

```diff
@@ -39,58 +39,63 @@ class SE0071Derived : SE0071Base {
   }
 }
 
-// SR-3043: Diagnostics when accessing deinit
+// https://github.com/apple/swift/issues/45633
+// Diagnostics when accessing deinit
 
-class SR3043Base {
-}
+class Base {}
 
-class SR3043Derived: SR3043Base {
+class Derived: Base {
   deinit {
     super.deinit() // expected-error {{deinitializers cannot be accessed}}
   }
 }
 
-let sr3043 = SR3043Derived()
-sr3043.deinit() // expected-error {{deinitializers cannot be accessed}}
-sr3043.deinit // expected-error {{deinitializers cannot be accessed}}
-SR3043Derived.deinit() // expected-error {{deinitializers cannot be accessed}}
+do {
+  let derived = Derived()
+  derived.deinit() // expected-error {{deinitializers cannot be accessed}}
+  derived.deinit // expected-error {{deinitializers cannot be accessed}}
+  Derived.deinit() // expected-error {{deinitializers cannot be accessed}}
+}
 
 // Allow deinit functions in classes
 
 class ClassWithDeinitFunc {
   func `deinit`() {
   }
 
-  func `deinit`(a: SR3043Base) {
+  func `deinit`(a: Base) {
   }
 }
 
-let instanceWithDeinitFunc = ClassWithDeinitFunc()
-instanceWithDeinitFunc.deinit()
-_ = instanceWithDeinitFunc.deinit(a:)
-_ = instanceWithDeinitFunc.deinit as () -> Void
-SR3043Derived.deinit() // expected-error {{deinitializers cannot be accessed}}
```

Comment:
The test case on this line is a dupe (see [old line 56](https://github.com/apple/swift/pull/60836/files#diff-1cd9a190598e3b2c93014976d719ec0203280751c5032152e7092ab1060d712eL56)); deleted it.